storage-proofs/src/circuit/drgporep.rs

```diff
@@ -566,7 +566,7 @@ mod tests {
         let sp = drgporep::SetupParams {
             drg: drgporep::DrgParams {
                 nodes,
-                degree,
+                degree: m,
```

Comment:
Either this is `m` or `degree`, but not `degree: m` please.

Reply:
Fair enough, I think Jake changed that because you didn't want the `degree - 1`.